metabase/metabase
# Add CTA to search dropdown

### Description

Adds a `View and filter all results` button on the Search bar dropdown to redirect users to SearchApp.

### How to verify

* Open the search dropdown, and type in a search. 
	* If there are results, there should be a `View and filter X results` footer at the bottom of the dropdown. Clicking on this footer should take you to the full page search.
	* If there are no results, there should not be a footer.

### Screenshots

State with search results
<img width="398" alt="image" src="https://github.com/metabase/metabase/assets/25306947/02697f63-c811-45fc-8d46-ebba7b497d32">

State with no search results
<img width="408" alt="image" src="https://github.com/metabase/metabase/assets/25306947/68b7e031-ef53-4597-aa97-d9d31bf2c97b">



### Checklist
